eng/Build.props

```diff
@@ -83,10 +83,22 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Build.Utilities.Core.15.1.1012.csproj" />
 
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Bcl.AsyncInterfaces.7.0.0.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.IO.Pipelines.7.0.0.csproj" />
```

Comment:
Any new packages should be grouped at the end of this file per this documentation - https://github.com/dotnet/source-build-reference-packages/pull/612/files#diff-366b25472963c6b33188d84139f6c8997298c5adc857a2624ab0d26a2d87baf0L15.  Not a big deal for this particular change but it is the recommended practice with a valid reason.

Reply:
Okay thanks. I tried to add them where I thought was logical. Will extract to the end.

Reply:
I already fixed it for you.